packages/adblocker-puppeteer-example/index.ts

```diff
@@ -59,5 +59,17 @@ function getUrlToLoad(): string {
     console.log('style', style.length, url);
   });
 
+  blocker.on('script-rule-matched', (rule, context) => {
+    console.log('script-matched', rule, context);
+  });
+
+  blocker.on('extended-rule-matched', (rule, context) => {
+    console.log('extended-rule-matched', rule, context);
+  });
+
+  blocker.on('style-rule-matched', (rule, context) => {
```

Comment:
I wonder if we could make it clearer in the naming that this is a hostname-based march of a cosmetic rule. Also, that it does not mean it marched in the page, just that it was selected by the engine as a candidate to be injected (I think it’s important to make this clear to avoid confusion)